app/controllers/provider/admin/backend_apis_controller.rb

```diff
@@ -42,7 +42,13 @@ def update
   end
 
   def destroy
-    # TODO
+    if @backend_api.backend_api_configs.any?
```

Comment:
I'd check directly for `backend_api.services`

Reply:
Is making `BackendApiConfig` as much transparent as possible the idea behind, @makikata?